frontend/app/actions/ActionRefiners.scala

```diff
@@ -105,6 +105,6 @@ object OptionEither {
     apply(Future.successful(x.map[Option[A]](Some.apply)))
 
   def liftFutureEither[A](x: Option[A]): OptionT[FutureEither, A] =
-    apply(Future.successful(\/.right(x)))
+    apply(Future.successful(\/.r[String](x)))
```

Comment:
scalaz is no longer covariant on `\/` so we have to make sure the right type parameter is used before passing into `apply`